### ISSUE TITLE
Panorama - empty backend GET variable when clicking on 'show details' from a pie chart object

### ISSUE THREAD

**Issue by TomGudman** [sni/Thruk#489]
Hi,

Setup: centos 5.11 + nagios 3.2.3 + thruk 1.88-4

When using panorama and clicking on a pie chart it does redirect me to the details view but it does not display anything because the 'backend' value is empty. In short the URL works if I remove: "backend=&'

Any idea how to remove backend or maybe set backend to the right value because otherwise I get a valid thruk page with no results unless I remove 'backend=&'.

I only have one backend. Thruk runs on the same box that our old nagios.

Reproduce: 
1. right click the pie chart, 
2. Click show details, 
3. Then it redirects me to http://$FQDN/thruk/#cgi-bin/status.cgi?backends=&hostgroup=$HOSTGROUPNAME

```
# grep backend /etc/thruk/*
thruk.conf:delay_pages_after_backend_reload = 0
thruk.conf:show_backends_in_table=0
thruk_local.conf:delay_pages_after_backend_reload = 0
thruk_local.conf:show_backends_in_table=0
```
Any ideas?

**Comment by TomGudman** [comment]
I recently installed thruk 1.88-4 on a old centreon (FAN 2.0 IIRC) and did not have the problem. So it must come from the configuration somewhere. If it is too hard to solve, don't worry just reject the issue I can live with it ;-)

**Comment by sni** [comment]
should be fixed now, thanks for your report

**Comment by TomGudman** [comment]
Thank you Sven for the awesome support!

**Comment by erSitzt** [comment]
Hi,

i had the same problem with `backend=&` and your fix solved this problem.
Now single hosts and services and service/host totals are working, but the host and service tables are not.

As it was somewhat related i did not open a new issue, hope thats ok.

![image](https://cloud.githubusercontent.com/assets/1625376/8523869/1244e8b8-23f8-11e5-80df-f7387c8b424c.png)

**Comment by sni** [comment]
should be fixed with 82904945ca

**Comment by erSitzt** [comment]
All good now, thanks Sven !
